state-chain/pallets/cf-ingress-egress/src/mock_eth.rs

```diff
@@ -109,15 +109,17 @@ impl NetworkEnvironmentProvider for MockNetworkEnvironmentProvider {
 	}
 }
 
-impl crate::Config for Test {
+impl_mock_runtime_safe_mode! { ingress_egress_ethereum: PalletSafeMode<Instance1> }
```

Comment:
Doing this would avoid having to add the `Instance1` in all the tests.

```suggestion
impl_mock_runtime_safe_mode! { ingress_egress_ethereum: PalletSafeMode<()> }
```

Reply:
(The actual value of `Instance` only matters if there are multiple instances in the runtime).